.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     name: Go Test
     strategy:
       matrix:
-        go-version: [1.18.x, 1.19.x, 1.20.x]
+        go-version: [1.20.x, 1.21.x, 1.22.x]
```

Comment:
Do we not want to use the same as in mojito and zerolog?

`go-version: [1.21.x, 1.22.x, 1.23.x]`

Reply:
Then I'll lift the go.mod version too